### DISCUSSION TITLE
Add tutorial 7: testing

### DISCUSSION THREAD

**Discussion by heitorPB** [encode/django-rest-framework#9443] [Ideas & Suggestions]
Testing is an important part of software development, but the tutorial does not cover that. I'd like to suggest adding another part about testing.

~~PS: I'm struggling with testing a DRF app~~

**Comment by martinberoiz** [comment]
Have you tried using the standard Django testing suite? I find it flexible enough for my testing needs.

**Reply by heitorPB** [reply]
Thanks for the suggestion, i'll look at it.

I know there are some docs in DRF and Django websites, but still, the DRF tutorial does not cover testing, hence my suggestion.

Should I create an Issue for this, instead of a discussion?

**Comment by browniebroke** [comment]
Agree with Martin, Django testing practices are very much applicable to DRF. The library provides some utilities that build on top of it, but that's a small layer of abstraction to make it more suitable for testing: https://www.django-rest-framework.org/api-guide/testing/
